### ISSUE TITLE
Source directories not 'ondemand' don't work

### ISSUE THREAD

**Issue by johrstrom** [OSC/ondemand-packaging#192]
I clone ondemand into `~/ondemand/src` and so this packaging library is building a `src-2.1.0.tar.gz` based off of the basename of the directory.  This is more for documentation than to actually fix, the behavior's likely fine and it's infact my side that needs to change.

Here are steps to replicate
```
git clone /tmp/src
cd /tmp/src
rake package:build['ubuntu-20.04']
```

and the error you get
```
== DEB build package=/work/ubuntu-20.04/src-2.1.0 ==
 dpkg-buildpackage -us -uc -ui
dpkg-buildpackage: info: source package ondemand
dpkg-buildpackage: info: source version 2.1.0
dpkg-buildpackage: info: source distribution UNRELEASED
dpkg-buildpackage: info: source changed by root <root@6ab11e8c7326>
 dpkg-source --before-build .
dpkg-buildpackage: info: host architecture amd64
 debian/rules clean
dh clean
   dh_clean
	rm -f debian/debhelper-build-stamp
	rm -rf debian/.debhelper/
	rm -f -- debian/ondemand.substvars debian/files
	rm -fr -- debian/ondemand/ debian/tmp/
	find .  \( \( \
		\( -path .\*/.git -o -path .\*/.svn -o -path .\*/.bzr -o -path .\*/.hg -o -path .\*/CVS -o -path .\*/.pc -o -path .\*/_darcs \) -prune -o -type f -a \
	        \( -name '#*#' -o -name '.*~' -o -name '*~' -o -name DEADJOE \
		 -o -name '*.orig' -o -name '*.rej' -o -name '*.bak' \
		 -o -name '.*.orig' -o -name .*.rej -o -name '.SUMS' \
		 -o -name TAGS -o \( -path '*/.deps/*' -a -name '*.P' \) \
		\) -exec rm -f {} + \) -o \
		\( -type d -a -name autom4te.cache -prune -exec rm -rf {} + \) \)
 dpkg-source -b .
dpkg-source: error: can't build with source format '3.0 (quilt)': no upstream tarball found at ../ondemand_2.1.0.orig.tar.{bz2,gz,lzma,xz}
dpkg-buildpackage: error: dpkg-source -b . subprocess returned exit status 25
debuild: fatal error at line 1182:
dpkg-buildpackage -us -uc -ui failed
rake aborted!
Command failed with status (29): [debuild --no-lintian --preserve-env ...]
/ondemand-packaging/gems/gems/ood_packaging-0.1.1/lib/ood_packaging/build.rb:270:in `block in debuild!'
/ondemand-packaging/gems/gems/ood_packaging-0.1.1/lib/ood_packaging/build.rb:269:in `chdir'
/ondemand-packaging/gems/gems/ood_packaging-0.1.1/lib/ood_packaging/build.rb:269:in `debuild!'
/ondemand-packaging/gems/gems/ood_packaging-0.1.1/lib/ood_packaging/build.rb:145:in `run!'
/ondemand-packaging/gems/gems/ood_packaging-0.1.1/lib/ood_packaging/tasks.rb:41:in `block (3 levels) in <top (required)>'
/ondemand-packaging/gems/gems/rake-13.0.6/exe/rake:27:in `<top (required)>'
/ondemand-packaging/rake:23:in `load'
/ondemand-packaging/rake:23:in `<main>'
Tasks: TOP => ood_packaging:package:build
```

**Comment by johrstrom** [comment]
You can close this as a wont fix, I don't mind -  I just wanted something to reference if I ever hit it or see it again.

**Comment by johrstrom** [comment]
I actually still get this error when I use `ondemand` as the basename. Could be some `podman` compatibility issue.

**Comment by treydock** [comment]
Is it possible to put entire output from failed command into this issue or maybe a gist or something?  I don't have podman on my Mac due to limitations with podman on Macs and don't have a readily available Ruby environment setup on OSC system with newer podman installed.

**Comment by treydock** [comment]
The lack of `orig.tar.gz` file means something broken before building, like section with this:

```
== Bootstrap DEB ==
```

Is when the code bootstraps the build environment in container based on tar ball generated outside the container into source tree.

**Comment by treydock** [comment]
Oh, one thing to try, remove quotes around dist.

```
rake package:build[ubuntu-20.04]
```

I have vague recollection that quotes in Rake arguments caused me problems

**Comment by treydock** [comment]
Fixed via #193
